chart/kubeapps/templates/apprepository-rbac.yaml

```diff
@@ -86,10 +86,7 @@ rules:
   resources:
   - apprepositories
   verbs:
-  - get
-  - create
-  - update
-  - delete
+  - "*"
```

Comment:
the `write` role was missing the `list` permissions

Reply:
So using "*" also adds `patch` and `watch`. Not sure if that's intended.

Reply:
yes, I mean, I would expect this role to be "admin" for apprepositories